iceberg/iceberg-handler/src/test/queries/positive/iceberg_major_compaction_unpartitioned_w_filter.q

```diff
@@ -29,7 +29,7 @@ create table ice_orc (
     last_name string
  )
 stored by iceberg stored as orc 
-tblproperties ('format-version'='2');
```

Comment:
I am starting to think if we should have extracted the default into Hive conf, WDYT?

Reply:
I don’t think Amoro defaults are bad. I added properties like ‘compactor.threshold.target.size’=‘1500’ in many major compaction tests because I wanted to enable major compaction on very small files.